ray-operator/apis/ray/v1/raycluster_types.go

```diff
@@ -221,19 +221,27 @@ type RayClusterStatus struct {
 	// +optional
 	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
 
-	// ReadyWorkerReplicas indicates how many worker replicas are ready in the cluster
```

Comment:
cc @davidxia @andrewsykim we will update the definition of "replica" in CRD status.